src/amo/components/Errors/ErrorComponent/index.js

```diff
@@ -0,0 +1,36 @@
+/* @flow */
+import * as React from 'react';
+import makeClassName from 'classnames';
+import NestedStatus from 'react-nested-status';
+
+import Card from 'ui/components/Card';
+
+import './styles.scss';
+
+type Props = {|
+  children: React.Node,
+  className?: string,
+  code: 400 | 401 | 404 | 500,
```

Comment:
This tells Flow that these are the valid values but, unfortunately, as we do not have complete Flow coverage in this app, we also need to check these values via code.

We often use `invariant` to do this, which you can see examples of in lots of other files. In this case we should define an `invariant` in the `render` function which will throw if `code` is not one of the valid values.

Reply:
Done :-)